docs/src/content/docs/fr/reference/configuration.mdx

```diff
@@ -568,3 +569,13 @@ starlight({
 ```
 
 Consultez la [référence des modules d'extension](/fr/reference/plugins/) pour plus de détails pour créer vos propres modules d'extension.
+
+### `credits`
+
+Permettre l'affichage d'un lien "Built with Starlight" dans le pied de page de votre site.
```

Comment:
```suggestion
Permet l'affichage d'un lien "Créé avec Starlight" dans le pied de page de votre site.
```

The feature was [translated](https://github.com/withastro/starlight/blob/main/packages/starlight/translations/fr.json#L31) live during Starlight birthday as there was multiple people speaking French in the call so we can safely use that.

Reply:
Oh of course!